packages/js78.rb

```diff
@@ -49,13 +47,13 @@ def self.patch
     system('for i in `cat debian/patches/series`; do  patch -p 1 < debian/patches/$i; done')
```

Comment:
Rubyize with `File.foreach 'debian/patches/series' {|i| system 'patch -p 1 < debian/patches/#{i}' }` ?

Reply:
That gives this error:
```
js78.rb:47:42: E: Lint/Syntax: unexpected token tLCURLY
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
    File.foreach 'debian/patches/series' {|i| system 'patch -p 1 < debian/patches/#{i}' }
                                         ^
js78.rb:47:54: E: Lint/Syntax: unexpected token tSTRING
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
    File.foreach 'debian/patches/series' {|i| system 'patch -p 1 < debian/patches/#{i}' }
                                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
js78.rb:102:1: E: Lint/Syntax: unexpected token $end
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
```

Reply:
Sorry, it should be `File.foreach 'debian/patches/series' {|i| system 'patch -p 1 < debian/patches/#{i}' }`

Reply:
Sorry, it should be `File.foreach 'debian/patches/series' {|i| system "patch -p 1 < debian/patches/#{i}" }`

Reply:
done

Reply:
Same error btw...

```
js78.rb:47:42: E: Lint/Syntax: unexpected token tLCURLY
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
    File.foreach 'debian/patches/series' {|i| system "patch -p 1 < debian/patches/#{i}" }
```